pa11y.json

```diff
@@ -0,0 +1,30 @@
+{
+    "defaults": {
+        "timeout": 10000
+    },
+    "chromeLaunchConfig": {
+        "args": [
+            "--no-sandbox",
+            "--disable-setuid-sandbox",
+            "--disable-dev-shm-usage",
+            "--disable-gpu",
+            "--headless"
+        ]
+    },
```

Comment:
From my experience, with pa11y-ci we won't need these settings (pa11y-ci probably has them by default)